website/docs/reference/widgets/list.md

```diff
@@ -115,10 +149,21 @@ Properties allow you to customize the widget, connect it to other widgets and tr
 | Property                   | Description                                                                                                                                                                                                                                               |
 | -------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | **Items**                  | Allows you to bind static or dynamic data collection to the widget.                                                                                                                                                                                       |
-| **Server-side Pagination** | Enables you to implement pagination by limiting the number of results fetched per Query request.  |
+| **Data Identifier** | Like `keys` in React, you need to select a data identifier from your dataset, which helps uniquely identify each item. This helps the List widget identify which items are added, have changed, or are removed. You can also combine two columns or data identifiers by enabling the `JS` mode. |
+| **Server-side Pagination** | Enables you to implement pagination by limiting the number of results fetched per query request.  |
```

Comment:
This field doesn't change anything in the query, here it says `by limiting the number of result`. Enabling this field does 2 things
1. Understands that the current data should not be paginated
2. Triggers a query when user clicks next or previous

Reply:
Updated. just linking to the section that shows how to serverside pagination